recipe/meta.yaml

```diff
@@ -126,3 +128,4 @@ extra:
     - sodre
     - xhochy
     - ocefpaf
+    - xylar
```

Comment:
If you think another maintainer would be helpful, I'm happy to contribute.  If you'd prefer not to have me, no problem.